site/src/components/button.md

```diff
@@ -126,28 +126,11 @@ Make a button take up the full width of a container.
 
 ### Loading buttons
 
-Ensure the user knows they need to wait for some event (like a fetch request) in order to interact with a button.
+Ensure the user knows they need to wait for some event (like a fetch request) in order to interact with a button. Can be used with the `disabled` attribute, as in the second button example below.
```

Comment:
I'm wondering if `is-disabled` should also be mentioned somewhere? Not just for buttons, since we have that modifier available on other components too.

Reply:
Updated this with a list of commonly used state-related classes